### ISSUE TITLE
Accessibility: Button in disabled mode has a to low color contrast

### ISSUE THREAD

**Issue by telion2** [telekom/scale#1139]
1. Open this url of Documentation: 
https://telekom.github.io/scale/?path=/docs/components-button--standard&globals=locale:en;colorMode:light
2. Set Disabled true
3. Get the colors of the button which are: 
```
 color: hsla(0, 0%, 0%, 0.4);   //translates to #999999
 background-color: #d0d0d2;
```
4. Open https://webaim.org/resources/contrastchecker/
5. Enter both hex values (#999999 &  #d0d0d2) 

**Current Behavior**
The Contrast-Checker does not pass the accessibility test, mentioning a to low contrast. 

**Expected Behavior**
The Contrast-Checker (or any other contrast checker) should pass the accessibility tests.

**Comment by JoschaIco** [comment]
Hi telion2, 

for disabled states there is always a conflict between visualizing something as disabled (low contrast) and the topic of accessibility. If you make the contrast too high it will not be recognized as disabled anymore. That's why the disabled state is exempt from the WCAG contrast minimum for text colors. You can find more information in the [WCAG guidelines](https://www.w3.org/TR/WCAG21/#contrast-minimum).

**Comment by JoschaIco** [comment]
> The visual presentation of [text](https://www.w3.org/TR/WCAG21/#dfn-text) and [images of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text) has a [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) of at least 4.5:1, **except for the following**:
> 
> Large Text
> [Large-scale](https://www.w3.org/TR/WCAG21/#dfn-large-scale) text and images of large-scale text have a contrast ratio of at least 3:1;
> 
> Incidental
> **Text or images of text that are part of an inactive [user interface component]**(https://www.w3.org/TR/WCAG21/#dfn-user-interface-components), that are [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
> 
> Logotypes
> Text that is part of a logo or brand name has no contrast requirement.
